apps/frontend/src/components/BlogNav.tsx

```
@@ -0,0 +1,143 @@
+
+                    <input
+                        type="checkbox"
+                        onChange={handleChange}
+                        id="search-toggle"
+                        ref={search}
+                        aria-expanded={search?.current?.checked || false}
+                        className="peer/search hidden"
+                    />
+                    <label
+                        htmlFor="search-toggle"
+                        role="checkbox"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<input type="checkbox">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)